lighthouse/tests/beacon_node.rs

```diff
@@ -1338,8 +1337,8 @@ fn enr_match_flag_over_ipv4() {
 fn enr_match_flag_over_ipv6() {
     const ADDR: &str = "::1";
     let addr = ADDR.parse::<Ipv6Addr>().unwrap();
-    let udp6_port = unused_udp6_port().expect("Unable to find unused port.");
-    let tcp6_port = unused_tcp6_port().expect("Unable to find unused port.");
+    let udp6_port = 0;
+    let tcp6_port = 0;
```

Comment:
same here. In principle this test, as it is right now, is testing that undesirable behaviour occurs